.gitignore

```diff
@@ -54,3 +54,6 @@ target/
 
 pom.xml.releaseBackup
 release.properties
+
+new_path/
+/new_path/
```

Comment:
This is probably not indended for master?

Reply:
The validation of the MAGE-TAB produces a folder called `new_path` is intended to be used by the CI GitHub Actions only, but I added it to the .gitignore to avoid someone by mistake to submit the folder.